src/pages/MovieList/styled.jsx

```diff
@@ -0,0 +1,233 @@
+import styled from "styled-components";
+import { ReactComponent as star } from "../../assets/star.svg";
+
+export const Container = styled.div`
+  margin: 0 auto;
+  display: flex;
+  margin-top: 50px;
+  flex-direction: column;
+  justify-content: center;
+`;
+
+export const Content = styled.div`
+  display: flex;
+  flex-direction: column;
+  margin: 0 auto;
+  align-self: center;
+  grid-gap: 10px;
+
+  @media (max-width: ${({ theme }) => theme.breakpoints.m}) {
+    align-items: center;
+  }
+
+  @media (max-width: ${({ theme }) => theme.breakpoints.s}) {
+    align-items: center;
+  }
+`;
+
+export const Header = styled.span`
+  display: flex;
+  align-self: flex-start;
+  font-size: 26px;
+  font-style: normal;
+  font-weight: 600;
+  line-height: 120%;
+
+  @media (max-width: ${({ theme }) => theme.breakpoints.m}) {
+    display: flex;
+    font-size: 24px;
+  }
+
+  @media (max-width: ${({ theme }) => theme.breakpoints.s}) {
+    display: flex;
+    font-size: 18px;
+  }
+`;
+
+export const TilesRow = styled.div`
+  display: grid;
+  grid-template-columns: auto auto auto auto;
+  grid-gap: 16px;
+
+  @media (max-width: ${({ theme }) => theme.breakpoints.m}) {
+    display: flex;
+    flex-wrap: wrap;
+  }
+
+  @media (max-width: ${({ theme }) => theme.breakpoints.s}) {
+    display: flex;
+    flex-direction: column;
+  }
+`;
+
+export const Tile = styled.div`
+  display: flex;
+  padding: 5%;
+  width: 324px;
+  border-radius: 5px;
+  flex-shrink: 0;
+  flex-direction: column;
+  background-color: ${({ theme }) => theme.colors.tile.background};
+  box-shadow: 0px 4px 12px 0px rgba(186, 199, 213, 0.5);
+
+  @media (max-width: ${({ theme }) => theme.breakpoints.s}) {
+    width: 360px;
+    height: 260px;
+    display: flex;
+    padding: 16px;
+    flex-direction: row;
+    align-items: flex-start;
+    gap: 16px;
+  }
+`;
+
+export const TileHeader = styled.header`
+  font-size: 22px;
+  @media only screen and (max-width: ${({ theme }) => theme.breakpoints.ms}) {
+    font-size: 16px;
+  }
+  @media only screen and (max-width: ${({ theme }) => theme.breakpoints.s}) {
+    font-size: 12px;
+  }
+`;
+
+export const Image = styled.img`
+  width: 100%;
+  height: 100%;
+  border-radius: 5px;
+  @media (max-width: ${({ theme }) => theme.breakpoints.s}) {
+    width: 50%;
+    height: 100%;
+  }
+`;
+
+export const Description = styled.div`
```

Comment:
Zmieniłbym nazwę tego komponentu na Content. Warto utworzyć osobny folder TileSmall (najlepiej jako podfolder w folderze Tile) i przenieść tam style komponentów użytych w nim. Oczyści to troche ten kod.